builder/qemu/config.go

```diff
@@ -161,6 +161,66 @@ func (c QemuSMPConfig) getMaxCPUs() int {
 	return totalVCPUs
 }
 
+// Booting in EFI mode
+//
+// Use these options if wanting to boot on a UEFI firmware, as the options to
+// do so are different from what BIOS (default) booting will require.
+type QemuEFIBootConfig struct {
+	// Boot in EFI mode instead of BIOS
+	//
+	// This is required for more modern guest OS
+	//
+	// If either or both of `efi_firmware_code` or `efi_firmware_vars` are
+	// defined, this will implicitely be set to `true`.
+	EnableEFI bool `mapstructure:"efi_boot" required:"false"`
+	// Path to the CODE part of OVMF (or other compatible firmwares)
+	//
```

Comment:
I would be mindful of the extra lines in the comments as they do show up in the generate docs.